corehq/apps/domain/models.py

```
@@ -906,12 +906,13 @@ class DomainAuditRecordEntry(models.Model):
     cp_n_saved_app_changes = models.BigIntegerField(default=0)
     cp_n_enterprise_console_exports = models.BigIntegerField(default=0)
     cp_n_enterprise_settings_edits = models.BigIntegerField(default=0)
+    cp_n_questions_locked = models.BigIntegerField(default=0)
 
     @classmethod
     @atomic
-    def update_calculations(cls, domain, property_to_update):
+    def update_calculations(cls, domain, property_to_update, count=1):
         obj, is_new = cls.objects.get_or_create(domain=domain)
```

Ruff (0.16.0)

[warning] Unpacked variable `is_new` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)